styleguide/backstop.json

```diff
@@ -32,7 +32,7 @@
   "scenarios": [
     {
       "label": "atoms-button",
-      "url": "http://localhost:3000/?p=atoms-button",
+      "url": "http://localhost:3000/?p=atoms-button.html",
```

Comment:
This is not the right URL, the paths change a bit more significantly than just adding `.html` when opening in a new window. Please check the URL using the "Open in a new Window" link that's available when clicking the Gear at the top right hand corner of pattern lab.